src/libs/ajax/ajax-override-utils.test.ts

```diff
@@ -0,0 +1,89 @@
+import { makeError, makeSuccess, mapJsonBody } from './ajax-override-utils';
+
+type FetchFn = typeof fetch;
+
+describe('mapJsonBody', () => {
+  it('updates response body', async () => {
+    // Arrange
+    const fetch: FetchFn = jest.fn().mockResolvedValue(new Response(JSON.stringify({ foo: 1, bar: 2, baz: 3 })));
+
+    const incrementValues = mapJsonBody((obj) =>
```

Comment:
This is kind of a micronit and maybe doesn't really jive with how `mapJsonBody` is intended to be used: but I stumbled a bit with the AAA organization here... Since `mapJsonBody` is the method under test, I was expecting it to appear in the `// Act` section.

While I appreciate the nicely named `incrementValues` it kinda distracted from the focus of the test, which is that that particular function is getting applied _by_ `mapJsonBody` -- is there another way to do this where `incrementValues` just becomes an inline/anonymous function (maybe with a comment explaining that it's just there to increment the results), and the real focus of the `// Act` is the actual `mapJsonBody` invocation

Similar nit for all the tests in this file.